src/main/scala/org/broadinstitute/hail/io/annotators/SampleFamAnnotator.scala

```diff
@@ -12,17 +12,24 @@ object SampleFamAnnotator {
   val numericRegex =
     """^-?(?:\d+|\d*\.\d+)(?:[eE]-?\d+)?$""".r
 
-  def apply(filename: String, delim: String, isQuantitative: Boolean, missing: String,
+  val famID = "famID"
```

Comment:
why split these out?  Having the strings constructed as literals in the Struct constructor makes it very easy to look at the schema.